proto/echo/v1/echo.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package echo.v1;
+option go_package = "foxygo.at/pb/echo";
+import "google/api/annotations.proto";
+
+// Echo service.
+service EchoService {
+  // Hello greets.
+  rpc Hello (HelloRequest) returns (HelloResponse) {
+    option (google.api.http) = { post:"/api/echo/hello" body:"*" };
```

Comment:
Do we need this option for this repo? Seems like it might be confusing to new users of the repo since we are not doing anything with http.

Getting rid of it means we also get rid of the proto vendoring.

Reply:
well i kinda actually wanted to keep this here on purpose. I thought longer term it would be also neat to have HTTP apis mocked.

but also: I think it is good to test that transitive dependency closures and slightly more tricky protos with options are worky. can i convince you?

Reply:
If you did it deliberately, then ok. Not knowing that, it seemed like unnecessary complexity.